app/controllers/api/backyards_controller.rb

```diff
@@ -11,10 +13,30 @@ def show
     render json: backyard_article, serializer: BackyardsShowSerializerSerializer, root: :backyard_article
   end
 
+  def create
+    backyard_article = build_backyard_article
+    if backyard_article.persisted?
+      render json: { message: 'Your backyard article has been successfully created!' }, status: 201
+    else
+      render json: { error_message: 'Please fill in all required fields' }, status: 422
+    end
+  end
+
   private
 
   def get_country
     country_response = Geocoder.search([params[:lat], params[:lon]])
     country_response.first.country
   end
+
+  def article_params
+    params[:backyardArticle].permit(:title, :theme, :location, body: [])
+  end
+
+  def build_backyard_article
+    backyard_article = current_user.articles.build(article_params)
+    backyard_article['backyard'] = true
+    backyard_article.save
+    backyard_article
```

Comment:
what dose this do?

Reply:
line 40

Reply:
we have to return the backyard article after we update it. The .save method only returns a boolean. And ruby only returns the latest line of code, so we have to call on the article again.